pandas/core/tools/datetimes.py

```diff
@@ -89,6 +93,7 @@
 DatetimeScalar = Union[Scalar, datetime]
 
 DatetimeScalarOrArrayConvertible = Union[DatetimeScalar, ArrayConvertible]
+DictConvertible = Union[Dict[str, List], "DataFrame"]
```

Comment:
I'm not sure whether python 3.8 allows using dict and list in this case (python 3.9 does): `Union[dict[str, list], "DataFrame"]`

Reply:
python 3.8 doesn't allow `Union[dict[str, list], "DataFrame"]`  I tried it.